src/multi_vector_simulator/A0_initialization.py

```diff
@@ -403,7 +403,11 @@ def process_user_arguments(
 
     if pdf_report is True:
         user_input.update(
-            {"path_pdf_report": os.path.join(path_output_folder, PDF_REPORT)}
+            {
+                "path_pdf_report": os.path.join(
```

Comment:
Is `path_pdf_reprot` no constant variable?

Reply:
it should be. In that case I found that because this is used as argument for the function `def print_pdf(app=None, path_pdf_report=os.path.join(OUTPUT_FOLDER, "out.pdf"))` setting a global variable `ARG_PATH_REPORT_PDF = "path_pdf_report"` would not hint that `ARG_PATH_REPORT_PDF` is related to the argument named `path_report_pdf` in the function. Because it is only used in two modules (and in test of these modules), and therefore less prone to typo mistake that are getting out of hand I was hesitant to do the change.

Reply:
Hm, okay. I think at some point we should replace this (why not `PATH_REPORT_PDF`), but we can leave it for now.

Reply:
I am not implementing this one in this PR